input/docs/writing-builds/reusing-builds.md

```diff
@@ -0,0 +1,82 @@
+Order: 80
+Description: How to reuse build code in different projects and repositories
+---
+
+Cake supports different ways to share build code across multiple projects and repositories.
+
+| Method                         | Sharing of classes                              | Sharing of aliases                              | Sharing of tasks                                |
+|--------------------------------|-------------------------------------------------|-------------------------------------------------|-------------------------------------------------|
+| [Recipe NuGet packages]        | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-times" style="color:red"></i>   | <i class="fa fa-check" style="color:green"></i> |
+| [Addins]                       | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-times" style="color:red"></i>   |
+| [Assemblies in NuGet packages] | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-times" style="color:red"></i>   | <i class="fa fa-times" style="color:red"></i>   |
```

Comment:
Are we missing a link for this one?

Reply:
This actually works due to some magic in Wyam. But I added link explicitely.

Reply:
Gotcha.  I think adding it explicitly makes a lot of sense.